project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/docgen_from_Jira.yaml

```diff
@@ -0,0 +1,61 @@
+name: docgen from Jira
+on:
+  workflow_dispatch
+
+jobs:
+   generate:
+     runs-on: ubuntu-latest
+     steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # full history for metadata
+          submodules: false
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Python install
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install "lander<2.0.0"
+
+      - name: condaenv
+        run: |
+          conda init bash
+          conda create --name docsteady-env docsteady -c lsst-dm -c conda-forge
+        shell: bash
+
+      - name: docugen
+        env:
+          JIRA_USER: {% raw %}${{ secrets.JIRA_USER }} {% endraw %}
+          JIRA_PASSWORD: {% raw %}${{ secrets.JIRA_PASSWORD }} {% endraw %}
+        run: |
+          source ${CONDA}/bin/activate docsteady-env
+          docsteady generate-tpr {{ cookiecutter.plan }}  {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex
+          echo `ls -la {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex`
+        shell: bash
+ 
+      - name: Create commit
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git commit -m "Update from JIRA " {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex 
```

Comment:
Maybe remove the trailing space or replace it with a period.

As I understand it, the intention here is that the commit happens to a branch that the user has already created and provides to the workflow when triggering it.

Just a thought: in the long run, having the action automatically open a PR (perhaps with a branch name specified as a parameter) would be nice so that the result can be checked before being published.

Reply:
For the PR I think most users (these are not necessarily developers) would disagree and say having a PR is tedious and extra steps ..  we can ask ..